app/src/test/java/by/academy/jc/ht/zabrodsky/ht9/MyStackTest.java

```diff
@@ -0,0 +1,90 @@
+package by.academy.jc.ht.zabrodsky.ht9;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+public class MyStackTest {
```

Comment:
Missing test cases where stack is iterable (used in for-each constructs, for example)